po/LINGUAS

```diff
@@ -1,6 +1,7 @@
 be
 be@latin
 bg
+cs_CZ
```

Comment:
Please name this just "cs" -- we should avoid country specific variants in most cases, except for a few known cases (en, es, and zh).